### PR TITLE
Adapt unittest for read_txt_Renishaw() function

### PR DESCRIPTION
**Pull request by sangttruong**

### CODE REVIEW

File: R/read_txt_Renishaw.R [hunk 1]
```diff
@@ -164,48 +164,47 @@ read_txt_Renishaw <- function(file = stop("file is required"),
...
-    tmp <- read_txt_Renishaw("fileio/txt.Renishaw/laser.txt.gz", "ts")
+    tmp <- read_txt_Renishaw(laser, "ts")
     expect_equal(dim(tmp), c(nrow = 84L, ncol = 3L, nwl = 140L))
     expect_equal(colnames(tmp), c("t", "spc", "filename"))
   })
 
   test_that("map (= default)", {
-    skip("TODO: adapt to new package")
-    tmp <- read_txt_Renishaw("fileio/txt.Renishaw/chondro.txt", "xyspc")
+    tmp <- read_txt_Renishaw(chondro, "xyspc")
```
Comment (GegznaV) [inline R/read_txt_Renishaw.R]: Do not use `chondro` as it was removed from `hyperSpec`. Use `faux_cell` instead (in this and all other places where `chondro` is used)

**Review by GegznaV — CHANGES_REQUESTED** [review]

File: R/read_txt_Renishaw.R (hunk 1 above)
Comment (sangttruong) [reply R/read_txt_Renishaw.R]: There is no such testing file called faux_cell in the testing data folder.

Comment (GegznaV) [reply R/read_txt_Renishaw.R]: Sorry, I was confused by the name `chondro` as many years it meant a different thing.

Please add `f_` to variable names that contain file paths, e.g. `f_chondro`

File: R/read_txt_Renishaw.R [hunk 2]
```diff
@@ -164,48 +164,47 @@ read_txt_Renishaw <- function(file = stop("file is required"),
 
 hySpc.testthat::test(read_txt_Renishaw) <- function() {
   context("read_txt_Renishaw")
+  path <- system.file("extdata/fileio/txt.Renishaw", package="hySpc.read.txt")
+  paracetamol <- paste0(path, "/paracetamol.txt")
+  laser <- paste0(path, "/laser.txt.gz")
+  chondro <- paste0(path, "/chondro.txt")
```
Comment (GegznaV) [inline R/read_txt_Renishaw.R]: Can you, please, not use the names `paracetamol`, `laser`, and `chondro` for file names. Use  `f_paracetamol`, `f_laser`, and `f_chondro` instead.

**Review by GegznaV — CHANGES_REQUESTED** [review]
As I see the paths to files were changed and the tests now work.
Please, change the names of variables with file names.

**Comment by GegznaV** [comment]
I think, for "**Adapt** unittest for read_txt_Renishaw() function" PR these changes would be OK. But after merging, "**Improve** unit test for read_txt_Renishaw() function" PR should be created and each imported object should be tested using this template:
https://github.com/r-hyperspec/hySpc.read.txt/issues/55#issue-962095045

**Review by GegznaV — APPROVED** [review]

**Pull request merged**